README.md

```diff
@@ -724,6 +758,31 @@ I18n.translations["pt-BR"] = {
 }
 ```
 
+## Using YAML file as config store
+Earlier versions of this gem was mainly using YAML config file for loading file.  
+And it was changed to have Ruby object as core configuration implementation.  
+For those using YAML config file and don't want to convert it into an initialzer,
+A loader class is created for loading such file.
+YAML config file will be supported for `3.x`, but might deprecated and removed in next major release.
```

Comment:
I don't agree with this. YAML should have first-class support along with Ruby. It shouldn't be "maybe deprecated".